conf/reef/nvmeof/ceph_nvmeof_4-nvmeof-gw_scale_cluster.yaml

```diff
@@ -0,0 +1,50 @@
+globals:
+  - ceph-cluster:
+      name: ceph
+      vm-size: ci.standard.large
```

Comment:
this might dont work. Need to check with CI team.

Reply:
But we have it in all test confs, ref any conf here  - https://github.com/red-hat-storage/cephci/tree/master/conf/reef/nvmeof

Not sure how this affects